sensor/kubernetes/certrefresh/testutils/testutils.go

```diff
@@ -0,0 +1,33 @@
+package testutils
```

Comment:
Oh, cmon, testutils inside testutils?

If you need a file with test helpers, then the following should be done:
1. Place it in a file `testing_test.go`, so that it is visible for the testing code.
2. Make all helper functions require the `t *testing.T` object in the input. That would prevent accidental usage outside of the test code.

Or alternatively, change to using test suite. That would create a natural place for the test helpers.

Reply:
Problem is that I'm using these functions in different packages. Since they are very few and quite basic, an alternative would be to simply repeat them in their respective _test.go files. Do you have a better idea?

Reply:
I see you found a solution :) That is great!